dev/insights/galaxy_ng.env

```diff
@@ -1,9 +1,7 @@
-PULP_CONTENT_PATH_PREFIX=/api/automation-hub/v3/artifacts/collections/
```

Comment:
Why this parameter got removed. Removing it breaks `download_url` returned by `api/automation-hub/v3/collections/<namespace>/<name>/versions/<version>/` endpoint.